src/templates/filter-bar-controller.js

```diff
@@ -29,12 +32,20 @@ FilterBarController.prototype.selectFilter = function(filterName) {
 };
 
 FilterBarController.prototype.getFilteredNotes = function(notes) {
-    const activeTab = Filters[getFilterName(store.getState())];
+    const state = store.getState();
+    const filterName = getFilterName(state);
+    const activeTab = Filters[filterName];
     if (!notes || !activeTab) {
         return [];
     }
 
-    return notes.filter(activeTab().filter);
+    const filteredNoteReads = getFilteredNoteReads(state);
```

Comment:
Yeah I saw your note(heh) about that in slack. We don't need the list anywhere else, so I'll add a selector for just checking if the note id is in the list 👍